build-tools/packages/build-cli/src/library/repoPolicyCheck/fluidBuildTasks.ts

```diff
@@ -227,7 +230,8 @@ async function eslintGetScriptDependencies(
 			const configFile = fs.readFileSync(eslintConfig, "utf8");
 			config = JSON5.parse(configFile);
 		} else {
-			config = import(`file://${path.resolve(eslintConfig)}`) as EslintConfig;
+			// This code assumes that the eslint config will be in CommonJS, because if it's ESM the require call will fail.
+			config = require(path.resolve(eslintConfig)) as EslintConfig;
```

Comment:
Remove the .mjs support option from above - or better make that it's own case that always fails

Reply:
Done in latest.